afs/media/js/views/components/reports/physical-thing.js

```diff
@@ -371,6 +386,137 @@ define([
+                        const resourceDomainConceptIds = resource.domains.map(x => x.conceptid)
+                    
+                        // If there is an IIIF manifest in the related resources, load the thumbnail from that manifest
+                        if (resourceDomainConceptIds.includes(IIIFManifestConceptId)) {
+                            const imageResource = await self.getThumbnail(resource);
+                            if (imageResource) {
+                                self.imageSummary(Array({
+                                    displayname: imageResource.label,
+                                    thumbnail: imageResource.sequences[0].canvases[0].thumbnail['@id']
+                                }));
+                            }
+                        }
+                    });
+                }
+            };
+
+            loadRelatedResources();
+
+            const statmentData = self.resource()?.statement;
+            if (statmentData) {
+                self.statementsSummary(statmentData.map(x => {
+                    const type = self.getNodeValue(x, 'statement_type');
+                    const content = self.getNodeValue(x, 'statement_content');
+                    const language = self.getNodeValue(x, 'statement_language');
+                    const tileid = self.getTileId(x);
+                    return { type, content, language, tileid }
+                }));
+            };
+
+            const typeData = self.resource()?.type;
+            if (typeData) {
+                self.typeSummary(Array({
+                    type: self.getNodeValue(typeData, '@display_value')
+                }));
+            };
+
+            const identiferData = self.resource()?.identifier;
+            if (identiferData) {
+                self.identifierSummary(identiferData.map(x => {
+                    const type = self.getNodeValue(x, 'identifier_type');
+                    const content = self.getNodeValue(x, 'identifier_content');
+                    const tileid = self.getTileId(x);
+                    return { type, content, tileid }
+                }));
+            };
+
+            const ownerData = self.resource()?.['current owner'];
+            if (ownerData) {
+                self.ownerSummary(ownerData['instance_details'].map(x => {
+                    const displayValue = self.getNodeValue(x, 'display_value');
+                    const link = self.getResourceLink({resourceId: self.getNodeValue(x, 'resourceId')});
+                    return { displayValue, link }
+                }));
+            };
+
+            const dimensionData = self.resource()?.dimension;
+            if (dimensionData) {
+                self.dimensionsSummary(dimensionData.map(x => {
+                    const value = self.getNodeValue(x, 'Dimension_value ');
+                    const unit = self.getNodeValue(x?.dimension_unit, '@display_value');
```

Comment:
you should be able to specify x, 'dimension_unit' here.  Avoid requesting @display_value directly.